.travis.yml

```diff
@@ -143,11 +148,13 @@ script:
   #   /home/travis/.travis/functions: line 109: ulimit: core file size: cannot modify limit: Operation not permitted
   - |
     ulimit -c unlimited || :
+  - du -d2 -h $DOCKER_VOLUME_PREFIX
```

Comment:
Do we need this?
We can see similar information by ccache statistics report: https://travis-ci.com/github/pitrou/arrow/jobs/477453957#L1533

Reply:
No, that was to debug until I got it to work.